en/docs/security-announcements/cloud-security-bulletins/asgardeo/2025/Asgardeo-2025-H1.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Additionally, to map this document, 
1. Add an `index.md` file to maintain all the listing related to 2025 (refer to sample at https://github.com/wso2/docs-security/blob/main/en/docs/security-announcements/cve-justifications/2025/index.md)
2. Need to update `mkdocs.yml` file to map all the pages (refer to CVE justification related references as a sample : https://github.com/wso2/docs-security/blob/main/en/mkdocs.yml#L347)

Reply:
Yes please. 

[2025 Security Bulletins](https://github.com/wso2/docs-security/blob/main/en/docs/security-announcements/cloud-security-bulletins/asgardeo/%7B%7B#base_path#%7D%7D/security-announcements/cloud-security-bulletins/asgardeo/2025) link is not working for now since we did not have this index.md 

Please add that file similar to https://github.com/wso2/docs-security/blob/main/en/docs/security-announcements/cve-justifications/2025/index.md

Reply:
Could you please also add this to mkdocs.yml similar to what is done here to get the menu working https://github.com/wso2/docs-security/pull/117/files#diff-25b25bdf9af26b2bd51113914da350cc25a45c8d2c8a75cc3549ea8034e85de6

Why I'm trying to get all this done here is to make sure we can use the PR as an example for creating future bulletins. Thanks!

Reply:
Fixed with [16371c9](https://github.com/wso2/docs-security/pull/121/commits/16371c9cb14d5b1ddb634deccce5e10bcbfce1d7)